client/ayon_houdini/plugins/publish/collect_karma_rop.py

```diff
@@ -28,7 +29,13 @@ def process(self, instance):
 
         rop = hou.node(instance.data.get("instance_node"))
 
-        default_prefix = evalParmNoFrame(rop, "picture")
+        try:
+            default_prefix = evalParmNoFrame(rop, "picture")
```

Comment:
Should this also be `self.evalParmNoFrame`?

Reply:
https://github.com/ynput/ayon-houdini/pull/190/commits/d92689e719a3147c23b59843ace486d627894aeb